Data/Text/Format/Heavy/Types.hs

```diff
@@ -1,9 +1,17 @@
-{-# LANGUAGE ExistentialQuantification, TypeFamilies, FlexibleContexts, OverloadedStrings #-}
+{-# LANGUAGE ExistentialQuantification, TypeFamilies, FlexibleContexts, OverloadedStrings, CPP #-}
 -- | This module contains basic type definitions
 module Data.Text.Format.Heavy.Types where
 
 import Data.Default
-import Data.Monoid
+
+#if MIN_VERSION_base(4,9,0)
+import Data.Monoid (Monoid)
+import Data.Semigroup ((<>))
+import qualified Data.Semigroup as Semigroup
+#else
+import Monoid
```

Comment:
Hm. This surprises me a bit. Does there exist a module named "Monoid" in some versions? Maybe you meant "Data.Monoid"?

Reply:
Hmm, whoops. Yeah, that's an accident. I'll edit the commit and force-push